web/src/app/shared/converters/loi-converter/loi-data-converter.ts

```diff
@@ -82,7 +82,7 @@ export class LoiDataConverter {
         jobId,
         geometry: {
           coordinates: new GeoPoint(geometry.coord.x, geometry.coord.y),
-          type: geometry.geometryType,
+          type: GEOJSON_GEOMETRY_TYPES.get(geometry.geometryType),
```

Comment:
Could you please add a TODO to refactor these geometry conversions into geometry-converter.ts?